node/node.go

```diff
@@ -32,31 +33,41 @@ type Options struct {
 	GenesisEpoch uint64
 	// max slots for duty to wait
 	DutySlotsLimit uint64
+	Context        context.Context
 }
 
 // Node represents the behavior of SSV node
 type Node interface {
 	// Start starts the SSV node
-	Start(ctx context.Context) error
+	Start() error
+	// InformObserver informs observer
```

Comment:
@nivBlox comment doesn't explain anything

Reply:
also, does we absolutely need to add those to the interface? I'd imagine we could have a dedicated interface for that that the node implements

Reply:
@nivBlox we actually have that interface in Observer

Reply:
"ssvNode" is implementation of "Node" so need the "Node" to implement it too

Reply:
@alonmuroch so why do we need the Node interface?

Reply:
@nivBlox I don't think we need the node interface, we can open an issue to remove it.

Reply:
https://github.com/ethereum/eth2-ssv/issues/87